File: app/src/main/java/com/communityx/adapters/GroupInfoAdapter.java
package com.communityx.adapters;

import android.content.Context;
import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;
import butterknife.BindView;
import butterknife.ButterKnife;
import com.communityx.R;
import com.communityx.utils.Utils;

public class GroupInfoAdapter extends RecyclerView.Adapter<GroupInfoAdapter.ViewHolder> {

    private Context mContext;
    private LayoutInflater mInflater;

    public GroupInfoAdapter(Context mContext) {
        this.mContext = mContext;
        mInflater = LayoutInflater.from(mContext);
    }

    @NonNull
    @Override
    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
        return new ViewHolder(mInflater.inflate(R.layout.item_group_info,viewGroup,false));
    }

    @Override
    public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
        if (i == 0) viewHolder.showAdmin(true);
    }

    @Override
    public int getItemCount() {
        return 3;
    }

    class ViewHolder extends RecyclerView.ViewHolder{

        @BindView(R.id.text_admin)
        TextView textAdmin;

        public ViewHolder(@NonNull View itemView) {
            super(itemView);
            ButterKnife.bind(this,itemView);
            textAdmin.setVisibility(View.GONE);
        }

        public void showAdmin(boolean show){
            Utils.showHideView(textAdmin,show);
        }
    }
}

File: app/src/main/java/com/communityx/adapters/GroupInfoAdapter.kt
package com.communityx.adapters

import android.content.Context
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import butterknife.BindView
import butterknife.ButterKnife
import com.communityx.R
import com.communityx.utils.Utils

class GroupInfoAdapter(private val mContext: Context) : RecyclerView.Adapter<GroupInfoAdapter.ViewHolder>() {

    private val mInflater: LayoutInflater = LayoutInflater.from(mContext)

    override fun onCreateViewHolder(viewGroup: ViewGroup, i: Int): ViewHolder {
        return ViewHolder(mInflater.inflate(R.layout.item_group_info, viewGroup, false))
    }

    override fun onBindViewHolder(viewHolder: ViewHolder, i: Int) {
        if (i == 0) viewHolder.showAdmin(true)
    }

    override fun getItemCount(): Int {
        return 3
    }

    inner class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

        @BindView(R.id.text_admin)
        lateinit var textAdmin: TextView

        init {
            ButterKnife.bind(this, itemView)
            textAdmin!!.visibility = View.GONE
        }

        fun showAdmin(show: Boolean) {
            Utils.showHideView(textAdmin!!, show)
        }
    }
}
